src/segment-loader.js

```diff
@@ -825,6 +826,7 @@ export default class SegmentLoader extends videojs.EventTarget {
     // BANDWIDTH_VARIANCE and add one so the playlist selector does not exclude it
     this.bandwidth =
       switchCandidate.playlist.attributes.BANDWIDTH * Config.BANDWIDTH_VARIANCE + 1;
+    this.playlist_.excludeUntil = Date.now() + ABORT_EARLY_BLACKLIST_MILLIS;
```

Comment:
Should we log a warning like we do when blacklisting other playlists? Related, should we change this to be an event that can notify `MasterPlaylistController` that it needs to blacklist the playlist so that all blacklisting goes through the same path?